src/pca-gophish-composition.service

```diff
@@ -8,16 +8,13 @@ AssertPathIsMountPoint=/var/pca/pca-gophish-composition/data
 [Service]
 Restart=always
 # Stop containers (if running) when unit is stopped
```

Comment:
I know it's not in this PR, but is this an accurate comment for an `ExecStartPre` declaration?

Reply:
Good catch!  I fixed that and cleaned up the other comments a bit in https://github.com/cisagov/pca-gophish-composition-packer/pull/44/commits/d59b63958924e1af9667b837a156ecae99fcd7eb.

Reply:
Excellent!